app2/src/main/java/com/example/recipebox/Contact.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.recipebox
+
+//java class database
+//users information
```

Comment:
Above classes and functions, use proper KDoc.
https://kotlinlang.org/docs/kotlin-doc.html#module-and-package-documentation